spec/spec_helper.rb

```diff
@@ -1,16 +1,98 @@
-# frozen_string_literal: true
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
What's with all these changes?

Reply:
I re-generated this. Will minimize changes